From: Bernhard M. Wiedemann <bwiedemann suse de>

   do not ask for ntp service in all protocols
   which is very slow (bnc#782060)
Index: libntp/decodenetnum.c
===================================================================
--- libntp/decodenetnum.c.orig	2015-10-21 18:13:49.000000000 +0200
+++ libntp/decodenetnum.c	2015-11-06 10:38:49.777763897 +0100
@@ -71,8 +71,9 @@
 		cp = name; 
 	}
 	ZERO(hints);
-	hints.ai_flags = Z_AI_NUMERICHOST;
-	err = getaddrinfo(cp, "ntp", &hints, &ai);
+	hints.ai_socktype = SOCK_DGRAM;
+	hints.ai_flags = Z_AI_NUMERICHOST | Z_AI_NUMERICSERV;
+	err = getaddrinfo(cp, "123", &hints, &ai);
 	if (err != 0)
 		return 0;
 	INSIST(ai->ai_addrlen <= sizeof(*netnum));
